README.md

```diff
@@ -16,3 +16,33 @@ support unwinding that crosses FFI boundaries.
 
 [shepherds-blog]: http://smallcultfollowing.com/babysteps/blog/2019/09/11/aic-shepherds-3-0/
 [zulip-room]: https://rust-lang.zulipchat.com/#narrow/stream/210922-wg-ffi-unwind/topic/welcome/near/177543226
+
+## Roadmap and current status
```

Comment:
How would this section differ from `plan.md`? Is `plan.md` supposed to be for more "overarching" goals? (Or would we simply replace `plan.md` with the roadmap?)

Reply:
Probably we can just remove plan.md